test/distributed/org/apache/cassandra/distributed/test/SimpleReadWriteTest.java

```diff
@@ -104,118 +99,10 @@ public void coordinatorWriteTest() throws Throwable
         }
```

Comment:
@adelapena I was just noticing...is this test actually non-deterministic around the loop of `executeInternal()` queries? Why should the QUORUM write guarantee that the write will be immediately visible to node 1?

Reply:
Even worst, the `i` variable in the loop is not used, and only the first replica is checked three times. I have modified the test to verify that the row makes into at least two replicas, no matter which ones. I'm also removing the `read_repair` option since the tests doesn't seem related with read repair. Indeed, RR would only be triggered in the last query without further checks, and we have other tests for that in place.